### PR TITLE
Add intro slide and hide icon when incorrectly authenticated AB#15489

### PR DESCRIPTION
**Pull request by BronzBierd**
# Fixes or Implements AB#nnnnn

## Description
1. check for valid authentication
2. Add slide and new buttons
3. Updated Functional tests.

## Testing

- [ ] Unit Tests Updated
- [X] Functional Tests Updated
- [ ] Not Required

### Mobile:
image

### Web:
image

### testing for valid auth state:
image

### Functional tests:
image

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue [hunk 1]
```diff
@@ -235,4 +275,9 @@ export default class AppTourComponent extends Vue {
         }
     }
 }
+
+.skipFirst {
+    z-index: 100;
+    position: absolute;
+}
```
[the remark this answers is not in the document]
Comment (BronzBierd) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:282]: This is to allow the app start button to align in the center (modal) without pixel pushing the start button.

[the remark this answers is not in the document]
Comment (BronzBierd) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:282]: image

**Review by ytqsl — DISMISSED** [review]

File: Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (BronzBierd) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:281]: I just pushed to remove the margin left on the button its now centered correctly

**Review by ytqsl — DISMISSED** [review]

File: Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:281]: Can the .skipFirst class be removed?

[the remark this answers is not in the document]
Comment (BronzBierd) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:281]: sorry if you look carefully you'll see my picture isn't actually perfectly lined up, the start button has a ml-3 attached, that's what I removed.

The reason for `.skipFirst` class is the responsive nature of b-col. in order for  Start App Tour to be centered against the entire modal, will require additional.

one solution would be to use an empty b-col
<b-col> skip button
<b-col centered>
<b-col> empty

I don't quite like this cause we'll be squeezing when it comes to mobile view ports and this is guaranteed to work.

[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:281]: There are simpler solutions using flexbox.

[the remark this answers is not in the document]
Comment (BronzBierd) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue:281]: from what I can see from a source like https://css-tricks.com/snippets/css/a-guide-to-flexbox/, I would need the empty b-col.

[the remark this answers is not in the document]
Comment (MikeLyttle) [reply Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue]: This has been resolved using Bootstrap columns.

**Review by MikeLyttle — APPROVED** [review]

**Review by ytqsl — APPROVED** [review]
gotta love CSS

**Pull request merged**